content/tutorials/manage-the-rippled-server/configuration/configure-the-peer-crawler.md

```diff
@@ -0,0 +1,61 @@
+# Configure the Peer Crawler
+
+By default, `rippled` servers provide statistics publicly to anyone who asks using the [peer crawler API](peer-crawler.html), to make it easier to track the health and topology of the XRP Ledger's peer-to-peer network. You can configure your server to provide more or less information, or to reject peer crawler requests entirely. [New in: rippled 1.2.0][]
+
+This document contains steps for two options:
+
+- [Change the Information Reported by the Peer Crawler](#change-the-information-reported-by-the-peer-crawler)
+- [Disable the Peer Crawler](#disable-the-peer-crawler)
+
+## Change the Information Reported by the Peer Crawler
+
+To configure how much information your server provides in response to peer crawler requests, complete the following steps:
+
+1. Edit your `rippled`'s config file.
+
+        vim /etc/opt/ripple/rippled.cfg
+
+    {% include '_snippets/conf-file-location.md' %}<!--_ -->
+
+2. Add or update the `[crawl]` stanza in your config file, and save the changes:
+
+        [crawl]
+        overlay = 1
+        server = 1
+        counts = 0
+        unl = 1
+
+    The settings in this example represent the default values. A setting with a value of `1` means to share that type of information. A value of `0` means not to share that information. The names of the config fields match the names of the fields they control in the [peer crawler response](peer-crawler.html#response-format).
```

Comment:
Maybe `value of '1' means to include this data in the response` instead of `value of '1' means to share that type of information`. 

Just to make it clear that these just control the fields in the `crawl` response and nothing more. "Share" doesn't seem to fit right.